Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -600,6 +600,10 @@ void RenderBlockFlow::layoutBlock(bool relayoutChildren, LayoutUnit pageLogicalH
     if (state && state->pageLogicalHeight())
         setPageLogicalOffset(state->pageLogicalOffset(this, logicalTop()));
 
+    for (auto& descendant : m_descendantsNeedingTransformUpdate)
+        descendant->updateLayerTransform();
```

Comment:
This might have to happen before we compute layout overflow, so that the motion path-related transform on the child contributes to overflow on the container.

Reply:
Fixed.